### PR TITLE
Filter observations by site in the UI

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: modules/model/shared/src/main/scala/observe/model/package.scala [hunk 1]
```diff
@@ -38,6 +40,21 @@ extension (i: Instrument)
 //      case Instrument.Gpi   => true
 //      case Instrument.Ghost => false
 
+  def site: Option[Site] = i match
```
Comment (cquiroz) [inline modules/model/shared/src/main/scala/observe/model/package.scala:43]: Does iit need to be an option? instruments always have a site?

Comment (rpiaggio) [reply modules/model/shared/src/main/scala/observe/model/package.scala:43]: The `Instrument` enum has things like `Michelle`, `Visitor`, `Phoenix`, etc.

Comment (cquiroz) [reply modules/model/shared/src/main/scala/observe/model/package.scala:43]: @swalker2m Why do we have those ancient instruments, are we planning to port the old ocs data?

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**